Cesium.Compiler/stdlib/unistd.h

```diff
@@ -0,0 +1,2 @@
+__cli_import("Cesium.Runtime.UniStdFunctions::Sleep")
+int sleep(double);
```

Comment:
[Normally](https://www.man7.org/linux/man-pages/man3/sleep.3.html), `sleep` is `unsigned sleep(unsigned seconds)`. Are you sure we want `double` here? Yes, it looks more useful honestly, but in reality it can break compilation of certain programs.

Reply:
Hmm. It seems to be I found weird sample. it has `sleep(0.01)` and `void main()`. Let me look what on Windows.

Reply:
Sorry, are you still looking at that?

Reply:
Forget about this part.